filters/cropbyfocalpoint.go

```diff
@@ -0,0 +1,135 @@
+package filters
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando-stups/skrop/parse"
+	"github.com/zalando/skipper/filters"
+	"gopkg.in/h2non/bimg.v1"
+	"strconv"
+	)
```

Comment:
nitpicking: I am not sure if gofmt is working correctly here. The `)` should be at start of the line